data-prepper-core/src/test/resources/invalid_data_prepper_config_with_bad_keystore_password.yml

```diff
@@ -1,4 +1,4 @@
-serverPort: 1234
```

Comment:
We should probably duplicate these files (one for old and one for new) and add another integration test for each. That way we are ensuring the old behavior continues to work.

Reply:
I thought about it. I reasoned, adding files, which will soon get deleted is probably not worth it. Will add it now.

Reply:
I will change existing test cases to use snake_case and add only one test case to make sure camel case are accepted. There is no need to duplicate every test case to see camel case is still supported.

Reply:
Yea, since it has all the properties this works.